MoreLinq/ScanBy.cs

```diff
@@ -40,14 +40,17 @@ static partial class MoreEnumerable
         /// <returns>
         /// A sequence of keys paired with intermediate accumulator states.
         /// </returns>
-
```

Comment:
Please keep the blank line for clarity.

Reply:
I would argue that it reduces clarity (by adding separation between the documentation and the method), but I'll defer to you.